service/prometheus/spec.go

```diff
@@ -12,4 +12,5 @@ const (
 // PrometheusReloader represents a service that can reload Prometheus configuration.
 type PrometheusReloader interface {
 	Reload() error
+	RequestReload()
```

Comment:
Godoc would help understanding what is going on here when only seeing the interface. I couldn't write the godoc now without checking the code again.

Reply:
Legit. Updated.